tests/jenkins/run_test.sh

```diff
@@ -17,7 +17,7 @@ if [ $user == 'root' ]
 then
     python setup.py install || exit 1
 else
-    python setup.py install --prefix ~/.local || exit 1
+    python setup.py install --user || exit 1
```

Comment:
@lxn2 I over looked this. Why did you add the installation here?

@sxjscience Please remove them all together